packages/terra-date-time-picker/src/DateTimeUtils.js

```diff
@@ -195,6 +195,22 @@ class DateTimeUtils {
   static convertDateTimeStringToMomentObject(date, time, dateformat, hasSeconds) {
     return DateTimeUtils.updateTime(DateUtil.createSafeDate(DateUtil.convertToISO8601(date, dateformat)), time, hasSeconds);
```

Comment:
There are a couple of occurrences of createSafeDate in here that need to be switched to use the one in DateTimeUtil.

Reply:
Should be replaced here https://github.com/cerner/terra-framework/pull/851/commits/fa0f2d77abb55b9a6d36ebec85eb3201ca1a3381